### PR TITLE
[hydra-helm-chart] - Added preStop and postStart hooks in hydra deployment.yaml

### PR DESCRIPTION
**Pull request by namannagar89**
JIRA: https://livspaceengg.atlassian.net/browse/DO-38

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the backstage documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: charts/hydra/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for Ory Hydra
 name: hydra
-version: 1.31.0
+version: 1.31.1
```
Comment (ankit-ls) [inline charts/hydra/Chart.yaml:4]:
```suggestion
version: 1.32.0
```

Comment (namannagar89) [reply charts/hydra/Chart.yaml]: done

File: charts/hydra/values.yaml [hunk 2]
```diff
@@ -83,6 +83,18 @@ env:
   - name: LOG_LEVEL
     value: info
 
+containerSpec:
+  lifecycle:
+    enabled: false
+    # preStop: {}
+#       exec:
+#         command: ["sleep","10"]
+    postStart: {}
```
[the remark this answers is not in the document]
Comment (namannagar89) [reply charts/hydra/values.yaml]: commented.

File: charts/hydra/values.yaml [hunk 3]
```diff
@@ -83,6 +83,21 @@ env:
   - name: LOG_LEVEL
     value: info
 
+containerSpec:
+  lifecycle:
+    enabled: true
+    preStop:
+      exec:
+        command: ["/bin/sh", "-c", "sleep 10"]
```
Comment (ankit-ls) [inline charts/hydra/values.yaml:91]: This is not required to be enabled in default values, as this chart is being used by other apps as well. Let's keep this disabled and all values commented here.

Comment (namannagar89) [reply charts/hydra/values.yaml]: sure. disabled and commented out the preStop.

**Pull request merged**